src/components/RouteErrorBoundary/RouteErrorBoundary.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * RouteErrorBoundary
+ */
+
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import ErrorBoundary from '@folio/stripes-components/lib/ErrorBoundary';
+
+const RouteErrorBoundary = ({ children, escapeRoute, moduleName, isSettings }) => {
+  const intl = useIntl();
+  const history = useHistory();
+  const currentRoute = history.location.pathname;
+  const shouldGoToAppHome = currentRoute === escapeRoute;
+  let buttonLabel = intl.formatMessage({ id: 'stripes-core.routeErrorBoundary.goToAppHomeLabel' });
```

Comment:
We can simplify this section a bit to reduce calls made on `intl.formatMessage()`:

```
let buttonLabel;

 if (moduleName) {
    if (isSettings) {
      buttonLabel = 'stripes-core.routeErrorBoundary.goToModuleSettingsHomeLabel';
    } else if {
      buttonLabel = 'stripes-core.routeErrorBoundary.goToModuleHomeLabel';
    }
  } else {
    buttonLabel = 'stripes-core.routeErrorBoundary.goToAppHomeLabel';
}
```

Then you can put the following on line 36 (test this part out to make sure the moduleName part is okay):
`resetButtonLabel={intl.formatMessage({ id: buttonLabel}, moduleName && { name: moduleName })}`

Reply:
Good catch, @ryandberger! I made the change. 👍